app/api/v2/admin/users.rb

```diff
@@ -119,6 +119,22 @@ def search(field, value)
               users.all.tap { |q| present paginate(q), with: API::V2::Entities::User }
             end
 
+            desc 'Returns array of users with pending documents as paginated collection',
+                 security: [{ "BearerToken": [] }],
+                 failure: [
+                     { code: 401, message: 'Invalid bearer token' }
+                 ]
+            params do
+              optional :page,     type: Integer, default: 1,   integer_gt_zero: true, desc: 'Page number (defaults to 1).'
+              optional :limit,    type: Integer, default: 100, range: 1..1000, desc: 'Number of users per page (defaults to 100, maximum is 1000).'
+            end
+            get '/pending' do
+              users = User.joins(:labels).where(labels: { key: 'document', value: 'pending' })
```

Comment:
Can you post resulting SQL query pls

Reply:
`SELECT  `users`.* FROM `users` INNER JOIN `labels` ON `labels`.`user_id` = `users`.`id` WHERE `labels`.`key` = 'document' AND `labels`.`value` = 'pending' LIMIT 100 OFFSET 0`